setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setuptools.setup(
```

Comment:
[note]
- Consider including a license
- Consider including keywords
- When you package this on pypi, you'll want to set `zip_safe=True`

Reply:
I'm reading about the `zip_safe` flag - I'm getting the impression that I do *not* want `zip_safe=True`, because I use `__file__` to find the yaml datafiles?
https://setuptools.readthedocs.io/en/latest/setuptools.html#setting-the-zip-safe-flag

Reply:
license and keywords added in https://github.com/mvernacc/material-properties-interchange/pull/1/commits/284de5322bbad553f62134f539dc1e5423cf9fbe